app/src/api/ApiCalendar.js

```diff
@@ -0,0 +1,288 @@
+  }
+  /**
+   * Create Calendar event
+   * @param {string} calendarId for the event.
+   * @param {object} event with start and end dateTime
+   * @param {string} sendUpdates Acceptable values are: "all", "externalOnly", "none"
+   * @returns {any}
+   */
+  createEvent(event, calendarId = this.calendar, sendUpdates = "none") {
+    if (this.gapi) {
+      return this.gapi.client.calendar.events.insert({
+        calendarId: calendarId,
+        resource: event,
+        sendUpdates: sendUpdates,
+      });
+    } else {
+      console.log("Error: this.gapi not loaded");
+      return false;
+    }
+  }
+  /**
+   * Delete an event in the calendar.
+   * @param {string} eventId of the event to delete.
+   * @param {string} calendarId where the event is.
+   * @returns {any} Promise resolved when the event is deleted.
+   */
+  deleteEvent(eventId, calendarId = this.calendar) {
+    if (this.gapi) {
+      return this.gapi.client.calendar.events.delete({
+        calendarId: calendarId,
+        eventId: eventId,
+      });
+    } else {
+      console.log("Error: gapi is not loaded use onLoad before please.");
+      return null;
+    }
+  }
+  /**
+   * @returns {any} Get the user's basic profile information. Documentation: https://developers.google.com/identity/sign-in/web/reference#googleusergetbasicprofile
+   */
+  getBasicUserProfile() {
+    if (this.gapi) {
+      return this.gapi.auth2
+        .getAuthInstance()
+        .currentUser.get()
+        .getBasicProfile();
+    } else {
+      console.log("Error: gapi is not loaded use onLoad before please.");
+      return null;
+    }
+  }
+  /**
+   * Update Calendar event
+   * @param {string} calendarId for the event.
+   * @param {string} eventId of the event.
+   * @param {object} event with details to update, e.g. summary
+   * @param {string} sendUpdates Acceptable values are: "all", "externalOnly", "none"
+   * @returns {any}
+   */
+  updateEvent(
+    event,
+    eventId,
+    calendarId = this.calendar,
+    sendUpdates = "none",
+  ) {
+    if (this.gapi) {
+      return this.gapi.client.calendar.events.patch({
+        calendarId: calendarId,
+        eventId: eventId,
+        resource: event,
+        sendUpdates: sendUpdates,
+      });
+    } else {
+      console.log("Error: gapi is not loaded use onLoad before please.");
+      return null;
+    }
+  }
+  /**
+   * Get Calendar event
+   * @param {string} calendarId for the event.
+   * @param {string} eventId specifies individual event
+   * @returns {any}
+   */
+  getEvent(eventId, calendarId = this.calendar) {
+    if (this.gapi) {
+      return this.gapi.client.calendar.events.get({
+        calendarId: calendarId,
+        eventId: eventId,
+      });
+    } else {
+      console.log("Error: gapi is not loaded use onLoad before please.");
+      return null;
+    }
+  }
+}
+let apiCalendar;
+try {
+  apiCalendar = new ApiCalendar();
+} catch (e) {
+  console.log(e);
+}
+export default apiCalendar;
```

Comment:
I'm guessing all the above code came from a library that Google wrote?  If so, I'm surprised you needed to include it in your `src` directory as opposed to just importing the library from a `node_modules` directory or something.  Let's chat about this when we pair, I'm not familiar with Google's calendar API but I'm curious if there's a way to keep application and library code separate.